avail-go/examples/go-examples/staking_validate/main.go

```diff
@@ -92,9 +97,9 @@ func main() {
 	}
 
 	nonce := uint32(accountInfo.Nonce)
-	o := types.SignatureOptions{
+	o := extrinsic.SignatureOptions{
```

Comment:
Use builder pattern. The user should not care about `rv` and `gnesishash` or nonce.